client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -106,7 +108,30 @@ const Header: FunctionComponent = () => {
 						title: siteTitle,
 					},
 					site_creation_flow: 'gutenboarding',
-					theme: `pub/${ selectedDesign?.slug }`,
+					...( selectedDesign && { theme: `pub/${ selectedDesign?.slug }` } ),
+				},
+				...( bearerToken && { authToken: bearerToken } ),
+			} );
+		},
+		[ createSite, currentDomain, selectedDesign, siteTitle, siteVertical ]
+	);
+
+	const handleCreateSiteForDomains = useCallback(
```

Comment:
Could `handleCreateSite` and `handleCreateSiteForDomains` share same options passed to `createSite()`?